biodatasets/euadr/euadr.py

```diff
@@ -0,0 +1,162 @@
+
+_LICENSE = "Elsevier user license"
+
+_URL = "https://raw.githubusercontent.com/EsmaeilNourani/Deep-GDAE/master/data/EUADR_target_disease.csv"
+
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+class EUADR(datasets.GeneratorBasedBuilder):
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    DEFAULT_CONFIG_NAME = "euadr_bigbio_kb"
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="euadr_source",
+            version=SOURCE_VERSION,
+            description="EU-ADR source schema",
+            schema="source",
+            subset_id="euadr",
+        ),
+        BigBioConfig(
+            name="euadr_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="EU-ADR simplified BigBio schema for named entity recognition and relation extraction",
+            schema="bigbio_kb",
+            subset_id="euadr",
+        ),
+    ]
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                                "ASSOCIATION_TYPE": datasets.Value("string"),
+                                "PMID": datasets.Value("int32"),
+                                "NUM_SENTENCE": datasets.Value("int32"),
+                                "ENTITY1_TEXT": datasets.Value("string"),
+                                "ENTITY1_INI": datasets.Value("int32"),
+                                "ENTITY1_END": datasets.Value("int32"),
+                                "ENTITY1_TYPE": datasets.Value("string"),
+                                "ENTITY2_TEXT": datasets.Value("string"),
+                                "ENTITY2_INI": datasets.Value("int32"),
+                                "ENTITY2_END": datasets.Value("int32"),
+                                "ENTITY2_TYPE": datasets.Value("string"),
+                                "SENTENCE": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+            
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+
+    def _split_generators(self, dl_manager):
+        urls = _URL
+        data_file = dl_manager.download_and_extract(urls)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "datafile": data_file
+                },
+            ),
+        ]
+
+    def _generate_examples(self, datafile):
+        key = 0
+        filepath = os.path.join(datafile)
+        datatable = pd.read_csv(filepath,sep='\t',encoding='latin',keep_default_na=False)
```

Comment:
Can you also check that the code formatting is ok?
```make check_file=biodatasets/euadr/euadr.py```

There are missing spaces in the `read_csv` function params, so it may be worth running `black`

```suggestion
        datatable = pd.read_csv(filepath, sep='\t', encoding='latin', keep_default_na=False)
```